interfaces/apparmor/backend_test.go

```diff
@@ -376,7 +376,7 @@ snippet
 func (s *backendSuite) TestCombineSnippets(c *C) {
 	restore := release.MockAppArmorLevel(release.FullAppArmor)
 	defer restore()
-	restore = osutil.MockMountInfo("") // mock away NFS detection
+	restore = osutil.MockMountInfo("") // mock away NFS/overlay detection
```

Comment:
Perhaps we want to offer mock helpers to mock the aspect (nfs, overlay) rather than the implementation detail and move those tests to osutil?